src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The `href` attribute in the `<link>` tag should point to a CSS file, not an SCSS file. Please ensure that the path is correct and points to a compiled CSS file.